synapseformation/client.py

```diff
@@ -50,8 +57,9 @@ def _create_synapse_resources(config: dict, creation_cls: SynapseCreation,
         parentid: Synapse folder or project id to store entities
     """
     if isinstance(config, dict) and config.get('type') == "Project":
-        project = creation_cls.get_or_create_project(name=config['name'])
-        parent_id = project.id
+        project_ent = creation_cls.get_or_create_project(name=config['name'])
+        _add_acl(creation_cls.syn, project_ent, config.get('acl', []))
```

Comment:
I would also add explicitly named parameters to this function to make it more readable 👀